corehq/util/metrics/prometheus.py

```diff
@@ -8,17 +8,49 @@
 
 
 class PrometheusMetrics(HqMetrics):
-    def __init__(self):
+    """Prometheus Metrics Provider"""
+
+    def initialize(self):
         self._metrics = {}
 
     def _counter(self, name: str, value: float = 1, tags: dict = None, documentation: str = ''):
+        """See https://prometheus.io/docs/concepts/metric_types/#counter"""
         self._get_metric(PCounter, name, tags, documentation).inc(value)
 
     def _gauge(self, name: str, value: float, tags: dict = None, documentation: str = ''):
+        """See https://prometheus.io/docs/concepts/metric_types/#histogram"""
         self._get_metric(PGauge, name, tags, documentation).set(value)
 
     def _histogram(self, name: str, value: float, bucket_tag: str, buckets: List[int], bucket_unit: str = '',
                   tags: dict = None, documentation: str = ''):
+        """
+        A cumulative histogram with a base metric name of <basename> exposes multiple time series
```

Comment:
what is `<basename>` referencing here? should it be `<name>` (i.e., the `name` argument) instead?